src/pages/MenuPage/MenuPage.tsx

```diff
@@ -5,18 +5,27 @@ import MenuItem from "../../components/MenuItem/MenuItem";
 import "../../index.css";
 import "./MenuPage.css";
 
-function MenuPage() {
-  const [items, setItems] = useState([]);
-  const [orders, setOrders] = useState([]);
-  const [visibleCount, setVisibleCount] = useState(6);
-  const [cartCount, setCartCount] = useState(0);
-  const [selectedCategory, setSelectedCategory] = useState("All");
+type Meal = {
+  id: string;
+  name: string;
+  image: string;
+  description: string;
+  category: string;
+  price: number;
+};
+
+function MenuPage(): React.ReactElement {
+  const [items, setItems] = useState<Meal[]>([]);
+  const [orders, setOrders] = useState<any[]>([]);
+  const [visibleCount, setVisibleCount] = useState<number>(6);
+  const [cartCount, setCartCount] = useState<number>(0);
+  const [selectedCategory, setSelectedCategory] = useState<string>("All");
 
   useEffect(() => {
     fetch("https://65de35f3dccfcd562f5691bb.mockapi.io/api/v1/meals")
       .then((res) => res.json())
       .then((data) => {
-        const formattedMeals = data.map((meal) => ({
+        const formattedMeals: Meal[] = data.map((meal: any) => ({
```

Comment:
couldn't any be replaced with Meal?
not a critical change but could be improved